app/src/main/java/org/mozilla/fenix/IntentReceiverActivity.kt

```diff
@@ -76,6 +76,7 @@ class IntentReceiverActivity : Activity() {
                 // original Intent (which might be a VIEW intent). However if there's no active browsing
                 // session then we do not want to re-process the Intent and potentially re-open a website
                 // from a session that the user already "erased".
+                intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
                 intent.flags and Intent.FLAG_ACTIVITY_LAUNCHED_FROM_HISTORY == 0
```

Comment:
I'm not sure if Kotlin has this operator, but shouldn't this be something like `intent.flags |= Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK`?

Reply:
It doesn't. https://stackoverflow.com/questions/23634970/what-is-kotlin-equivalent-for-bitwise-or-with-assignment

Reply:
Here's the proposal to add it to the language: https://github.com/Redrield/KEEP/blob/bitwise-operators/proposals/bitwise-operators.md

Reply:
Ah I see. Should we then do `intent.flags = intent.flags or Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK` since Intent.FLAG_ACTIVITY_LAUNCHED_FROM_HISTORY is getting overridden?

Reply:
Oh, right. Good call. I need to make sure that doesn't cause the original problem, but I'll try that.

Reply:
This is now moot with the updated fix.